ui/src/sandbox.rs

```diff
@@ -571,6 +584,72 @@ fn build_execution_command(
     cmd
 }
 
+fn parse_json_output(output: Vec<u8>) -> Result<(String, String)> {
+    let mut composed_stderr_string = String::new();
+    let mut composed_stdout_string = String::new();
+
+    let metadata_stream = cargo_metadata::Message::parse_stream(&output[..]);
```

Comment:
I'd have expected you could elide the `[..]`:

```suggestion
    let metadata_stream = cargo_metadata::Message::parse_stream(&output);
```